benchto-service/pom.xml

```diff
@@ -129,6 +129,10 @@
             <artifactId>hibernate-validator</artifactId>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>com.vladmihalcea</groupId>
```

Comment:
Is there a more official library?

Reply:
I cannot find the official library. Anyway, `hibernate-types-52` commonly used.

Reply:
I follow the author, he's pretty active in the Java community.